benchmark/start-server.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable */
```

Comment:
Should this benchmark be a ts file that is first compiled to a `.js` file?

Reply:
I tried but had some problems with URL imports. So let's do it in another PR later.